src/main/java/Duke.java

```diff
@@ -6,5 +8,50 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        // Greeting Message
+        System.out.println("Hello, I'm Duke!");
+        System.out.println("What can I do for you?");
+        System.out.println("\n");
+
+        Scanner sc = new Scanner(System.in);
+        int inputs = 0;
+        Task[] list = new Task[100];
+        while(true) {
+            String input = sc.nextLine();
+            // Exit Command
+            if(input.equals("bye")) {
+                System.out.println("Bye. Hope to see you again soon!");
+                break;
+            }
+            // List Command
+            else if(input.equals("list")) {
+                System.out.println("Here are the tasks in your list:");
+                for(int i = 0;i < inputs; i++) {
+                    System.out.print((i+1) + ". ");
+                    list[i].printTask();
+                }
+            }
+            else {
+                // Split into words to check whether done command is given
+                String[] words = input.split(" ");
+                if(words[0].equals("done")) {
+                    // Check if invalid done command is given
+                    if(words.length < 2) {
```

Comment:
I like that you checked if the command is valid

Reply:
Thanks Hui Hui!